vpd-tool/include/vpd_tool.hpp

```diff
@@ -200,5 +200,25 @@ class VpdTool
      * @return On success returns 0, otherwise returns -1.
      */
     int cleanSystemVpd() const noexcept;
+
+    /**
+     * @brief Dump all the inventory objects in JSON or Table format to console.
+     *
+     * This API dumps specific properties of all the objects in Phosphor
```

Comment:
> This API dumps specific properties of all the objects in Phosphor
     * Inventory Manager DBus tree to console in JSON or Table format to
     * console.


The statement is not correct to read. Also "T" is capital in table again.